src/button/borderless.scss

```diff
@@ -17,14 +18,18 @@
     padding: 0 $iui-sm;
   }
 
+  &[disabled] {
+    @include iui-button-disabled;
+  }
+
   // If icon is by itself, button takes 1:1 size ratio
-  > .iui-icon:only-child {
+  > .iui-button-icon:only-child {
```

Comment:
Unrelated to this PR, but I don't think this comment is correct. 40x38 is not 1:1

@FlyersPh9

Reply:
Oops, you are right.  We can remove / reword that comment.

Reply:
I was expecting you'd say we should fix the size to be 38x38 😆

Reply:
Removed comment